include/fastdds/dds/core/policy/QosPolicies.hpp

```diff
@@ -34,13 +34,14 @@
 #include <fastdds/rtps/common/Types.h>
 #include <fastdds/rtps/flowcontrol/FlowControllerConsts.hpp>
 #include <fastdds/rtps/resources/ResourceManagement.h>
-#include <fastrtps/types/TypeObject.h>
 #include <fastrtps/utils/collections/ResourceLimitedVector.hpp>
 
 namespace eprosima {
 namespace fastdds {
 namespace dds {
 
+using DomainId_t = uint32_t;
+
```

Comment:
This is going to be defined in another file. A PR is going to be issued creating this file. This PR should wait until that PR is merged.

Reply:
Waiting for https://github.com/eProsima/Fast-DDS/pull/4179